tests/helpers.py

```diff
@@ -1369,17 +1369,63 @@ def verify_volume_deleted_in_backend(interface, image_uuid, pool_name=None):
 
     try:
         ct_pod.exec_ceph_cmd(ceph_cmd=cmd, format='json')
-        return False
+        logger.info(
+            f"Verified: Volume corresponding to uuid {image_uuid} exists "
+            f"in backend"
+        )
+        return True
     except CommandFailed as ecf:
         assert valid_error in str(ecf), (
-            f"Error occurred while verifying volume is deleted in backend: "
+            f"Error occurred while verifying volume is present in backend: "
             f"{str(ecf)} ImageUUID: {image_uuid}. Interface type: {interface}"
         )
-    logger.info(
-        f"Verified: Volume corresponding to uuid {image_uuid} is deleted "
-        f"in backend"
-    )
-    return True
+        logger.info(
+            f"Volume corresponding to uuid {image_uuid} does not exist "
+            f"in backend"
+        )
+        return False
+
+
+def verify_volume_deleted_in_backend(
```

Comment:
this function should reside somewhere in ocs_ci/ dir as it could have usages that are not only in tests.
How about placing it in ocs_ci/ocs/resources/pvc.py?

Reply:
This function will use is_volume_present_in_backend function which is present in same module.
So if this function is moved to ocs_ci/ocs/resources/pvc.py, we will have to import tests.helpers.is_volume_present_in_backend .

The function is_volume_present_in_backend can be moved to ocs_ci/ocs/resources/pvc.py if it will satisfy the usage requirement for non-test module.
Other option is to move both the functions is_volume_present_in_backend and verify_volume_deleted_in_backend to ocs_ci/ocs/resources/pvc.py and modify test cases accordingly.